packages/cli-core/src/package-manager-controller/yarn_modern_package_manager_controller.ts

```diff
@@ -84,7 +84,7 @@ export class YarnModernPackageManagerController extends PackageManagerController
     await this.executeWithDebugLogger(
       targetDir,
       'yarn',
-      ['add', '-D', 'tsx', 'esbuild'],
+      ['add', '-D', 'corepack'],
```

Comment:
what is this being used for?

Reply:
It's for yarn https://yarnpkg.com/getting-started/install

But good question..it's supposed to be installed globally with node and it didn't complain before.

Let me check...

Reply:
removed f14f692c41